src/threesixty.js

```diff
@@ -101,11 +109,39 @@ var ThreeSixty = (function (window) {
         options.next = options.next || false;
         options.inverted = options.inverted || false;
 
-        container.style.width = options.width + 'px';
-        container.style.height = options.height + 'px';
-        container.style.backgroundImage = 'url("' + options.image + '")';
-        container.style.backgroundPosition = '0 0';
-        container.style.backgroundSize = (options.perRow * 100) + '%';
+        if(options.image) {
+            container.style.width = options.width + 'px';
+            container.style.height = options.height + 'px';
+        } 
+
+        self.loadImages = function() {
+            for (var i = 0; i < options.count; i++) {
```

Comment:
Can we omit the count parameter if array option is passed and simply use `options.array.length` instead?